### PR TITLE
✨ Add English course descriptions for communication, other engineering and general sections

### PR DESCRIPTION
**Pull request by EssamWisam**
Towards closing #13 

A quick review would be appreciated @Iten-No-404 . I prioritizied getting over with these in this week because I know school starts in about two weeks and widely announcing the website soon should be helpful. In all cases, take your time with the ones you are working on.

### CODE REVIEW

**Comment by Iten-No-404** [comment]
> A quick review would be appreciated @Iten-No-404 . 

On it. 
> take your time with the ones you are working on.

I was actually working on the Maths group of courses right now, which is last remaining group. Should be done in an hour or so.

**Comment by EssamWisam** [comment]
I think I couldn't remember the professor name(s) for the electrical engineering course and could not find myself in the classroom so I left that empty. Likewise, I didn't emphasize being detailed on the content for general courses such as Presentations and Technical Writing because I know they have changed and probably prone to change again.

File: public/department/General/Courses.yaml [hunk 1]
```diff
@@ -609,287 +609,359 @@
...
-                    \n The content of the course is described here...
+                    \n The course focuses on studying structures such as beams and frames and learning to determine their reaction at support points
+                    and whether they are stable or not. It also covers theoretical concepts related to building technology and cement.
                     \n
                     ### Project: 
-                    \n Information regarding the course project's goes here...  \n
+                    \n There is none.  \n
                     ### Material: 
-                    \n Material goes here...
+                    \n Nicely written lecture notes can be linked to here. You may find useful links in Older Google drives found in the 'Recently Joined' section.
```
Comment (Iten-No-404) [inline public/department/General/Courses.yaml:819]:
```suggestion
                    \n Nice [summaries](https://drive.google.com/drive/folders/1G3ojha70dX34ISYVAushBQKoA4X4D_wE) can be found here. You may find useful links in Older Google drives found in the 'Recently Joined' section.
```

File: public/department/General/Courses.yaml [hunk 2]
```diff
@@ -609,287 +609,359 @@
...
-                    \n Professor name goes here...    \n
+                    \n Dr. Nessreen ElAbiad    \n
                     ### Potential Prerequisites:
-                    \n Probable prerequisites go here... \n
+                    \n English \n
                     ### Course Content: 
-                    \n The content of the course is described here...
-                    \n
+                    \n Course equips students with the necessary technical skills to excel wherever such skills are needed in their
+                    business or personal lives.
```
Comment (Iten-No-404) [inline public/department/General/Courses.yaml:883]:
```suggestion
                    business or personal lives.\n
```

File: public/department/General/Courses.yaml [hunk 3]
```diff
@@ -609,287 +609,359 @@
...
           markdown: "### Instructor: 
-                    \n Professor name goes here...    \n
+                    \n Dr. Hesham    \n
                     ### Potential Prerequisites:
-                    \n Probable prerequisites go here... \n
+                    \n None \n
                     ### Course Content: 
-                    \n The content of the course is described here...
+                    \n The course explores concepts related to analyzing, evaluating and managing projects as well as the project's lifecycle,
+                     feasibility studies and dealing with failed projects. There is also an economics components focused on analyzing and understanding
```
Comment (Iten-No-404) [inline public/department/General/Courses.yaml:897]:
```suggestion
                     feasibility studies and dealing with failed projects. There is also an economics component focused on analyzing and understanding
```

File: public/department/General/Courses.yaml [hunk 4]
```diff
@@ -609,287 +609,359 @@
...
                     ### Material: 
-                    \n Material goes here...
-                    "  
+                    \n [Study notes](https://drive.google.com/drive/u/1/folders/1qrj8nj3eoJyW4nHh2ok4d8vAzzVI3LoS) and other nice material can be found in 2023 class Google drive. 
+                    You may also find useful links in Google drives found in the 'Recently Joined' section.
+                    " 
         - name: Electrical Engineering²
           image: https://engineering-jobs.theiet.org/getasset/85bead92-2e44-4692-b579-3ddf03a9a84b/
           markdown: "### Instructor: 
                     \n Professor name goes here...    \n
```
Comment (Iten-No-404) [inline public/department/General/Courses.yaml:843]:
```suggestion
                    \n 1st Part: Dr. Azza Nour El-Dien\n
                     2nd Part: Dr. Mohamed Elmashtoly (2021-present), Dr. Ahmed Saber (?-2020)\n
```

**Comment by EssamWisam** [comment]
> Very few typos, but overall spectacular work as always.✨ Please check the comments and approve what you see fit.

Approved everything. Sorry for the typos, thought the VSCode extension I have was working and I was using fewer brain cells than average while wokring on this. Thank you so much.

**Comment by EssamWisam** [comment]
For some reason the deploy previous doesn't work on my side; despite remember witnessing it work just a few weeks ago or something. Does this also happen to you?

**Comment by Iten-No-404** [comment]
> Sorry for the typos ... and I was using fewer brain cells than average while working on this. Thank you so much.

You're welcome. They weren't a lot anyway.



> For some reason the deploy previous doesn't work on my side; despite that I remember witnessing it work just a few weeks ago or something. Does this also happen to you?

I don't think so. I tried the #2 deployment now and it was working fine as well as the preview for this PR.

> Edit: Okay, it randomly works. Just like the site only randomly works when accessed from Egypt. Will probably change Netlify because I thought that an index file I added earlier solved this.

Okay, maybe create a new issue for this?

**Pull request merged**